OpenRA.Game/ModData.cs

```diff
@@ -161,16 +164,17 @@ public Map PrepareMap(string uid)
 			if (MapCache[uid].Status != MapStatus.Available)
 				throw new InvalidDataException($"Invalid map uid: {uid}");
 
+			// TO DO: Make loader use IMap interface instead of Map
```

Comment:
```suggestion
			// TODO: Make loader use IMap interface instead of Map
```
This is how TODO is written everywhere in the engine. Please change the 4 comments to this

Reply:
Fixed